server/socket.js

```diff
@@ -0,0 +1,86 @@
+import socketIO from'socket.io';
+import sticky from 'sticky-session';
+import http from 'http';
+import app from './server';
+import redisAdapter from 'socket.io-redis';
+import dotenv from 'dotenv';
+
+// Read .env file and set value in process.env
+dotenv.config()
+const DEFAULT_PORT = '8888'
+
+const listen = (expressApp, port = DEFAULT_PORT) => {
+  const io = socketIO();
+  const server = http.createServer(expressApp);
+
+  io.adapter(redisAdapter(process.env.REDIS_URL));
+  io.attach(server);
+  const isWorker = sticky.listen(server, port);
+
+  if (isWorker) {
+    const game = io.of('/game').on('connection', (socket) => {
```

Comment:
This block is rather large...

Reply:
I created files for each event handler in `server/socketHandlers` directory. But now I don't have test code for that. From now I will create Test code for those.

Reply:
Created tests for each socketHandler in `server/__tests__/socketHandlers` directory.